java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -135,4 +136,79 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> configResetStat(Route route);
+
+    /**
+     * Reads the configuration parameters of a running Redis server.<br>
+     * The command will be sent to a random node.
+     *
+     * @see <a href="https://redis.io/commands/config-get/">redis.io</a> for details.
+     * @param parameters An <code>array</code> of configuration parameter names to retrieve values
+     *     for.
+     * @return A <code>map</code> of values corresponding to the configuration parameters.
+     * @example
+     *     <pre>{@code
+     * Map<String, String> configParams = client.configGet(new String[] {"timeout" , "maxmemory"}).get();
+     * assert configParams.get("timeout").equals("1000");
+     * assert configParams.get("maxmemory").equals("1GB");
+     * }</pre>
+     */
+    CompletableFuture<Map<String, String>> configGet(String[] parameters);
+
+    /**
+     * Reads the configuration parameters of a running Redis server.
+     *
+     * @see <a href="https://redis.io/commands/config-get/">redis.io</a> for details.
+     * @param parameters An <code>array</code> of configuration parameter names to retrieve values
+     *     for.
+     * @param route Routing configuration for the command. Client will route the command to the nodes
+     *     defined.
+     * @return A <code>map</code> of values corresponding to the configuration parameters.<br>
+     *     When specifying a route other than a single node, it returns a dictionary where each
+     *     address is the key and its corresponding node response is the value.
+     * @example
+     *     <pre>{@code
+     * Map<String, String> configParams = client.configGet("timeout", Random).get().getSingleValue();
+     * assert configParams.get("timeout").equals("1000");
+     *
+     * Map<String, Map<String, String>> configParamsPerNode = client.configGet("maxmemory", ALL_NODES).get().getMultiValue();
+     * assert configParamsPerNode.get("<node1 address>").get("maxmemory").equals("1GB");
+     * assert configParamsPerNode.get("<node2 address>").get("maxmemory").equals("2GB");
+     * }</pre>
+     */
+    CompletableFuture<ClusterValue<Map<String, String>>> configGet(String[] parameters, Route route);
+
+    /**
+     * Sets configuration parameters to the specified values.<br>
+     * The command will be sent to all nodes.
+     *
+     * @see <a href="https://redis.io/commands/config-set/">redis.io</a> for details.
+     * @param parameters A <code>map</code> consisting of configuration parameters and their
+     *     respective values to set.
+     * @return <code>OK</code> if all configurations have been successfully set. Otherwise, raises an
+     *     error.
+     * @example
+     *     <pre>{@code
+     * String response = client.configSet(Map.of("timeout", "1000", "maxmemory", "1GB")).get();
+     * assert response.equals("OK")
```

Comment:
same examples has ';' at the end and some not, please align all

Reply:
Done!